app/templates/base.html

```diff
@@ -10,6 +10,16 @@
     <script src="{{ url_for('static', filename='htmx.min.js.gz') }}"></script>
     <!-- Load Plotly -->
     <script src="{{ url_for('static', filename='plotly.min.js.gz') }}"></script>
+
+    <!-- Use MathJax  -->
+    <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+    <script>
+        window.addEventListener('load', function() {
+            MathJax.typeset();
+        });
+    </script>
```

Comment:
This is a very big JS payload. You only use it on `index.html`. Consider moving this to the scripts block inside `index.html`.

That would be removing the above code and adding the following to index.html.
```
{% block scripts %}
    <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <script>
        window.addEventListener('load', function() {
            MathJax.typeset();
        });
    </script>
{% endblock %}
```